### DISCUSSION TITLE
Question(s) about Boa HTTP Server

### DISCUSSION THREAD

**Discussion by PerpSearch** [mrlt8/docker-wyze-bridge#1090] [Q&A]
I have a few cams that look like this is supported and have SD cards in place. I think I have the correct code in play since I show, 
 but I can't select anything to access the SD card contents.  My code looks like this. Am I missing a  line or two?

Thanks

version: '2.4.0'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
        - 1936:1935 # RTMP  (changed to 1936)
        - 8554:8554 # RTSP
        - 8888:8888 # HLS
        - 8889:8889 #WebRTC
        - 8189:8189/udp # WebRTC/ICE
        - 5050:5000 # WEB-UI (changed to 5050)
        environment:
         - IGNORE_OFFLINE=true
         - FPS_FIX=true
         - ENABLE_BOA=true
         - QUALITY=HD100
         - ROTATE_DOOR=true 

![2024-01-13_205307](https://github.com/mrlt8/docker-wyze-bridge/assets/92053831/f561bcf2-bc42-4ea9-aa9a-4dfa93cc15ce)

**Comment by g13092** [comment]
I never saw this BOA thing until now and you had me wondering.  I have the same 'view' on my setup, I can see the card is there, but no controls to access it.  I notice in the docs it says "NOTE: Boa has been disabled/removed in modern versions of the wyze firmware."  I'm guessing it's out of commission these days?
